alvr/session/src/settings.rs

```diff
@@ -218,6 +226,8 @@ pub struct VideoDesc {
 
     pub codec: CodecType,
 
+    pub rate_control_mode: RateControlMode,
```

Comment:
Do you think this setting should be advanced? This is something the user should not care too much about. CBR should still be the recommended setting and switching to VBR might get surprising results.

Reply:
I have a bit mixed feeling about this, but it makes sense. Can move it under advanced settings.